lib/gui/app/models/local-settings.js

```diff
@@ -17,13 +17,15 @@
 'use strict'
 
 const Bluebird = require('bluebird')
+const Storage = require('./storage')
 
 /**
  * @summary Local storage settings key
  * @constant
  * @type {String}
  */
 const LOCAL_STORAGE_SETTINGS_KEY = 'etcher-settings'
+const settingStorage = new Storage(LOCAL_STORAGE_SETTINGS_KEY)
```

Comment:
`settingsStorage` instead of `settingStorage` ?

Reply:
Yeah it's more correct, it just felt awkward to pronounce and type. I'll change it back.